api/src/middleware/competenciesRouter.ts

```diff
@@ -25,4 +27,25 @@ competenciesRouter.get('/', parsePaginationParams(), async (req, res, next) => {
   res.json(collectionEnvelope(competencies, competenciesCount));
 });
 
+competenciesRouter.post('/', async (req, res, next) => {
+  const { principalId } = req.session;
+  const { label: label, description: description } = req.body;
```

Comment:
Shorthands can be used here. There's no need to alias a property to a new name when the new name is the same as the old name.

Reply:
Good catch!